src/components/organisms/WorldStartForm/WorldStartForm.tsx

```diff
@@ -0,0 +1,198 @@
+  ...sidebarFooterProps
+}) => {
+  const worldId = world?.id;
+  const createMode = !worldId;
+  const { user } = useUser();
+
+  const defaultValues = useMemo(
+    () => ({
+      name: world?.name,
+      description: world?.config?.landingPageConfig?.description,
+      subtitle: world?.config?.landingPageConfig?.subtitle,
+      bannerImageFile: undefined,
+      bannerImageUrl: world?.config?.landingPageConfig?.coverImageUrl,
+      logoImageFile: undefined,
+      logoImageUrl: world?.host?.icon,
+    }),
+    [world]
+  );
+
+  const {
+    setValue,
+    reset,
+    watch,
+    formState: { dirty, isSubmitting },
+    register,
+    errors,
+    handleSubmit,
+  } = useForm({
+    mode: "onSubmit",
+    reValidateMode: "onChange",
+    validationSchema,
+    defaultValues,
+  });
+
+  const values = watch();
+
+  const [{ error, loading: isSaving }, submit] = useAsyncFn(async () => {
+    if (!values || !user) return;
+    if (!createMode || !worldId) return;
+
+    if (createMode) {
+      await createWorld(values, user);
+    } else {
+      await updateWorld({ ...values, id: worldId }, user);
+    }
+
+    reset(defaultValues);
+  }, [worldId, user, values, reset, createMode, defaultValues]);
+
+  if (error) {
+    console.error(WorldStartForm.name, error);
+  }
+
+  return (
+    <div className="WorldStartForm">
+      <Form onSubmit={handleSubmit(submit)}>
+        <AdminSidebarFooter
+          {...sidebarFooterProps}
+          saveButtonProps={{
+            type: "submit",
+            disabled: !dirty && !isSaving && !isSubmitting,
+            loading: isSubmitting || isSaving,
+          }}
+        />
+        <AdminSection>
+          <p>The URL of your world is:</p>
+          <p>
+            <span className="WorldStartForm__path-beginning">
+              {window.location.host}
+            </span>
+            <span className="WorldStartForm__path-end WorldStartForm__name">
+              {WORLD_ROOT_URL}/{values.name}
+            </span>
+          </p>
+        </AdminSection>
+        <AdminSection title="Name your world">
+          <AdminInput
+            label={
+              <>
+                What should we call this world?
+                <span className="WorldStartForm__name">*</span>
+              </>
+            }
+            name="name"
+            register={register}
+            errors={errors}
+          />
+        </AdminSection>
+        <AdminSection title="Add a subtitle">
+          <AdminInput
+            label="What should be the subtitle?"
+            name="subtitle"
+            register={register}
+            errors={errors}
+          />
+        </AdminSection>
+        <AdminSection title="Describe your world">
+          <AdminInput
+            label="How should we describe this world?"
+            name="description"
+            register={register}
+            errors={errors}
+          />
+        </AdminSection>
+        <AdminSection
+          title={
+            <>
```

Comment:
Inline prop, bad practice, causes rerender even when the rest of the props are the same.